admin/.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+  "plugins": [
+    ["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
```

Comment:
@yonikid15 [this solution](https://github.com/babel/babel/issues/11622#issuecomment-638609015) seems to work a bit better for getting rid of the command line warning:
```
{
  "plugins": [
  "@babel/plugin-proposal-class-properties",
  "@babel/plugin-proposal-private-methods",
  ]
}
```